pkg/controller/controller_utils.go

```diff
@@ -916,14 +930,20 @@ func AddOrUpdateTaintOnNode(c clientset.Interface, nodeName string, taint *v1.Ta
 // won't fail if target taint doesn't exist or has been removed.
 // If passed a node it'll check if there's anything to be done, if taint is not present it won't issue
 // any API calls.
-func RemoveTaintOffNode(c clientset.Interface, nodeName string, taint *v1.Taint, node *v1.Node) error {
+func RemoveTaintOffNode(c clientset.Interface, nodeName string, node *v1.Node, taints ...*v1.Taint) error {
+	if len(taints) == 0 {
+		return nil
+	}
 	// Short circuit for limiting amount of API calls.
 	if node != nil {
 		match := false
+	MATCHED:
```

Comment:
I don't like loop labels. Can you instead wrap `for _, taint...` into a annonymous function that returns 'match' and break if it returns true?